### PR TITLE
feat(ui): add toast component

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

- Add toast component
- Add a page level context to keep the global toast and loading status
- Update some svg

Mar-21-2022 14-39-22

## Linear Issue Reference

LOG-1772

## Testing

UT case added
test locally 

@logto-io/eng

### CODE REVIEW

File: packages/ui/src/components/AppContent/index.tsx [hunk 1]
```diff
@@ -11,10 +14,26 @@ export type Props = {
...
+    setToast('');
+  }, []);
+
   return (
-    <div className={classNames(styles.content, styles.universal, styles.mobile, styles[theme])}>
-      {children}
-    </div>
+    <PageContext.Provider value={context}>
+      <div
+        id="main"
```
Comment (wangsijie) [inline packages/ui/src/components/AppContent/index.tsx:30]: why not using a `<main>` tag?

File: packages/ui/src/components/Toast/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  if (!isVisible) {
+    return null;
+  }
+
+  return (
+    <div className={styles.toastContainer}>
+      <div
+        ref={toastElement}
+        className={styles.toast}
+        data-visible={show}
```
Comment (wangsijie) [inline packages/ui/src/components/Toast/index.tsx:51]: wondering if a `visibleRef` can do this?

**Review by wangsijie — APPROVED** [review]

File: packages/ui/src/components/AppContent/index.tsx (hunk 1 above)
Comment (simeng-li) [reply packages/ui/src/components/AppContent/index.tsx:30]: good call, BTW it will be used as querySelector for react-modal root element finding.

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Pull request merged**